pubkey_encrypt.info.yml

```diff
@@ -0,0 +1,8 @@
+name: Pubkey Encrypt
+description: Adds Password-based Public Key Encryption support.
+core: 8.x
+type: module
+package: Security
```

Comment:
I think this package will need to be "System" because "Security" does not ship with core